action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -1041,9 +1034,7 @@ func (stateDB *StateDBAdapter) CommitContracts() error {
 		}
 		copy(k[:], addrBytes)
 		v := stateDB.preimages[k]
-		h := make([]byte, len(k))
-		copy(h, k[:])
-		_, err = stateDB.sm.PutState(v, protocol.NamespaceOption(PreimageKVNameSpace), protocol.KeyOption(h))
+		_, err = stateDB.sm.PutState(v, protocol.NamespaceOption(PreimageKVNameSpace), protocol.KeyOption(k[:]))
```

Comment:
inside `KeyOption`, it will make a new slice and copy from input slice, so `h` here is not needed